src/map/clif.cpp

```diff
@@ -1169,7 +1169,7 @@ static void clif_set_unit_idle( struct block_list* bl, bool walking, send_target
 		p.maxHP = -1;
 		p.HP = -1;
 	}
-	p.isBoss = ( bl->type == BL_MOB && ( ( ( TBL_MOB *)bl )->db->mexp > 0 ) ) ? 1 : 0;
+	p.isBoss = ( bl->type == BL_MOB ) ? status_has_mode( &( ( mob_data *)bl )->db->status, MD_MVP ) : false;
```

Comment:
Is this value must be 2 for MVP monster (Golden skull logo)
and 1 for Miniboss (Silver skull logo) ?

Reply:
Fixed in fe559fe